### ISSUE TITLE
How to get the 3d proposal from 2d?

### ISSUE THREAD

**Issue by zhaobaozi** [smallcorgi/3D-Deepbox#16]
hi @smallcorgi i wonder which code is about get the 3d proposals for 2d?thanks

**Comment by smallcorgi** [comment]
Hi @zhaobaozi 

I only implemented the CNN part of the paper and you only can get the 3D dimension and orientation of  3d proposals.

In this project, I didn't implement the paper proposed "Correspondence Constraints".
If you want to get the location value (3D object location x,y,z), you can see the method of paper's supplementary materials.
They had some assumptions and used SVD decomposition to get the 3D object location.

**Comment by zhaobaozi** [comment]
Thanks!So the the paper proposed "Correspondence Constraints" only can get the (x,y,z) of proposals?the (w,h,l) and orientation only come from the CNN part? @smallcorgi

**Comment by tjufan** [comment]
@zhaobaozi   yeah, you should use the dimension, R_y， K to plot 3D Bounding Box
